frontend_tests/node_tests/pm_list.js

```diff
@@ -160,26 +160,24 @@ test("update_dom_with_unread_counts", (override) => {
 
```

Comment:
The node tests here are actually failing in this commit and fixed by these changes, but `unread_count` class is added to these elements being tests in later commits. Maybe it makes sense to squash them?